operations/mimir-rules-action/action.yaml

```diff
@@ -0,0 +1,14 @@
+name: mimirtool-sync
+author: jtlisi
```

Comment:
Should we change the author to Grafana? Even though Jacob was the original creator he no longer works at the company and this was created during his time here.

Reply:
Ops, yes, it definitely slipped.